pkg/networkpath/traceroute/tcp/tcpv4_unix.go

```diff
@@ -20,6 +20,19 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+var (
+	sendPacketFunc    = sendPacket    // for testing
+	listenPacketsFunc = listenPackets // for testing
+)
+
+type (
+	rawConnWrapper interface {
+		SetReadDeadline(t time.Time) error
+		ReadFrom(b []byte) (*ipv4.Header, []byte, *ipv4.ControlMessage, error)
+		WriteTo(h *ipv4.Header, p []byte, cm *ipv4.ControlMessage) error
+	}
+)
```

Comment:
This was moved from the `utils_unix.go` to here as we start using this interface here now to make testing easier.